apps/site/test/site_web/controllers/trip_compare_controller_test.exs

```diff
@@ -870,19 +230,4 @@ defmodule SiteWeb.TripCompareControllerTest do
              }
     end
   end
-
-  describe "Date and time selector" do
-    test "renders a date and time selector", %{conn: conn} do
-      conn = get(conn, trip_compare_path(conn, :index))
-      rendered = html_response(conn, 200)
-
-      # check there are date and time selectors:
-      refute Floki.find(rendered, "div[id=\"plan-time-select\"]") == []
-      refute Floki.find(rendered, "select[id=\"plan_date_time_hour\"]") == []
-      refute Floki.find(rendered, "select[id=\"plan_date_time_minute\"]") == []
-      refute Floki.find(rendered, "select[id=\"plan_date_time_am_pm\"]") == []
```

Comment:
Same question re: axing this whole file

Reply:
I initially removed the files which caused some test failures(still happening). Removing these altogether does seem clearer.